jrcc-document-access-libs/src/main/java/ca/bc/gov/open/jrccaccess/libs/TransactionInfo.java

```diff
@@ -56,10 +58,12 @@ public String getFileName() {
 	public LocalDateTime getReceivedOn() {
 		return receivedOn;
 	}
+
+	public String getUUIDStr(){return this.uuid.toString();}
 	
 	@Override
 	public String toString() {		
-		return	MessageFormat.format("Transaction sent from [{0}] on [{1}], fileName [{2}]", this.sender, this.receivedOn.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME), this.fileName);
+		return	MessageFormat.format("Transaction[{3}] sent from [{0}] on [{1}], fileName [{2}]", this.sender, this.receivedOn.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME), this.fileName,getUUIDStr());
```

Comment:
Good job on updating the to string, does this.id works? Prefered format with the lowercase with -

Reply:
yes. this.id works.
the default format is lowercase with -
sample: Transaction[4b9d6c87-d358-470b-937d-5a3c18cdcf1a] sent from [peggy] on [2019-09-17T09:44:51.479], fileName [jrcc-document-api.iml]